ui/src/App/Login/LoginPage.tsx

```diff
@@ -80,6 +80,7 @@ function LoginPage(): JSX.Element {
 						setErrorMessages([]);
 					}}
 					readOnly={isLoading}
+					validated={false}
```

Comment:
If I saw this prop without any context, I'd assume it was a prop to pass down dynamically if its been validated or not. Maybe change it to "showValidation" or "disableValidation"?

Reply:
using `validateInput`